app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -0,0 +1,68 @@
+module ContainerImageHelper
+  module TextualSummary
+    #
+    # Groups
+    #
+
+    def textual_group_properties
+      items = %w(name tag id)
+      items.collect { |m| send("textual_#{m}") }.flatten.compact
+    end
+
+    def textual_group_relationships
+      items = %w(containers registry ems)
+      items.collect { |m| send("textual_#{m}") }.flatten.compact
+    end
+
+    #
+    # Items
+    #
+
+    def textual_name
+      {:label => "Name", :value => @record.name}
+    end
+
+    def textual_tag
+      {:label => "Tag", :value => @record.tag}
+    end
+
+    def textual_id
+      {:label => "Image Id", :value => @record.image_ref}
+    end
+
+    def textual_containers
+      num_of_containers = @record.number_of(:containers)
+      label = ui_lookup(:tables => "containers")
+      h     = {:label => label, :image => "container", :value => num_of_containers}
+      if num_of_containers > 0 && role_allows(:feature => "containers")
+        h[:link]  = url_for(:action => 'show', :id => @record, :display => 'containers')
+        h[:title] = "View #{label}"
+      end
+      h
+    end
+
+    def textual_registry
+      registry = @record.container_image_registry
+      return nil if registry.nil?
+      label = ui_lookup(:table => "container_image_registry")
+      h = {:label => label, :image => "container_image_registry", :value => registry.host}
+      if role_allows(:feature => "container_image_registry_show")
+        h[:title] = "Show parent #{label} '#{registry.host}'"
+        h[:link]  = url_for(:controller => 'container_image_registry', :action => 'show', :id => registry)
+      end
+      h
+    end
+
+    def textual_ems
```

Comment:
please check https://github.com/ManageIQ/manageiq/pull/3402/
You can use ContainerSummaryHelper to reuse this code